metatron/src/main/java/uk/gov/ida/eidas/metatron/domain/EidasConfig.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.ida.eidas.metatron.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+
+public class EidasConfig {
+    @JsonProperty
+    private List<EidasCountryConfig> countries = new ArrayList<>();
+
+    @JsonIgnore
+    private Map<String, EidasCountryConfig> countriesMap;
+
+    public Collection<EidasCountryConfig> getCountries() {
+        return countries;
+    }
+
+    public EidasCountryConfig getCountry(String country) {
```

Comment:
This doesn't seem to be used either - the resolver service just pulls the countries from the `countries` list - can it be dropped?